featomic/build.rs

```diff
@@ -0,0 +1,47 @@
+#![allow(clippy::field_reassign_with_default)]
+
+use std::path::PathBuf;
+
+fn main() {
+    if cfg!(feature = "c-api") {
+        let crate_dir = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+
+        let generated_comment = "\
+/* ============    Automatically generated file, DOT NOT EDIT.    ============ *
+ *                                                                             *
+ *    This file is automatically generated from the featomic sources,          *
+ *    using cbindgen. If you want to make change to this file (including       *
+ *    documentation), make the corresponding changes in the rust sources.      *
```

Comment:
Can we give a link to the rust file here?

Reply:
done